XenAdmin/Dialogs/AssignLicenseDialog.resx

```diff
@@ -685,7 +685,7 @@
     <value>525, 201</value>
   </data>
   <data name="editionsGroupBox.TabIndex" type="System.Int32, mscorlib">
-    <value>2</value>
+    <value>0</value>

```

Comment:
This should stay 2 because the Groupbox sits inside the main container and there is the top label and the server planel before it.